include/camp/resource/hip.hpp

```diff
@@ -177,15 +181,15 @@ namespace resources
           switch (ma) {
             case MemoryAccess::Unknown:
```

Comment:
Did you remove MemoryAccess::Detect?

Reply:
I changed the name so it made more sense when used as both "detect" and "I can't figure out what this is." Unknown is detect, and will auto-detect how to free the thing when passed to deallocate.